docs/maintainers/adr/0004-remove-webjar-dependency.md

```diff
@@ -0,0 +1,86 @@
+use the minified CSS and JS bundles provided as part of the now self-published `hmrc-frontend` webjar via `hmrcHead` and `hmrcScripts`. 
+These bundles include the CSS and JS from both govuk-frontend and hmrc-frontend. For teams using this approach, the only
+assets still being retrieved from the govuk-frontend webjar are the handful of icons referenced
+in `govukTemplate` and `govukHeader`.
+
+Given the relative stability of these images, should we 
+* inline the images into this repository, for teams directly using `govukTemplate` or `govukHeader`
+* remove the govuk-frontend webjar dependency and
+* provide a mechanism to allow play-frontend-hmrc to supply its own images via the `assetsPath` parameter in 
+`govukTemplate` and `govukHeader` that would eliminate the need to add the `govuk-frontend` asset routing rule in
+  consuming microservices?
+
+## Decision Drivers
+
+* The need to make improvements and upgrades to govuk-frontend
+  available in play-frontend-govuk quickly
+* The increasing user base of play-frontend-govuk, and accelerating demand for new features and
+  improvements.
+* The desire to reduce boilerplate in consuming services.
+* The high number of services still referencing `lib/govuk-frontend/govuk/all.js`
+* The fact that the images have not changed for a long time (since September 2018)
+* The need to minimise the impact of breaking changes on service teams.
+* The hardship, frustration and toil the current manual publishing process is causing the team. 
+
+## Considered Options
+
+* Option 1: Do nothing
+* Option 2 (non-breaking): Inline the images and provide override mechanism but keep govuk-frontend dependency for now, advising
+  teams to move over to `hmrcHead` and `hmrcScripts` because a breaking change will be coming
+* Option 3 (breaking): Inline the images, provide override mechanism and remove govuk-frontend dependency
+
+## Decision Outcome
+
+Chosen option: Option 2, because doing so will (a) put is in a better position to eliminate the use of the webjar eventually,
+(b) allow us to make changes to play-frontend-hmrc to eliminate the `/govuk-frontend` route and (c) not introduce any
+breaking changes.
+
+## Pros and Cons of the Options
+
+### Option 1
+
+* Bad, because it doesn't improve the situation
+* Good, because no further work is required
+
+### Option 2
+
+* Good, because we can remove the need to include the govuk-frontend route when integrating with play-frontend-hmrc
+* Good, because teams using `hmrcHead` and `hmrcScripts` no longer use any assets from the govuk-frontend webjar
+* Good, because it should accelerate the adoption of the minified bundles via `hmrcHead` and `hmrcScripts`, increasing
+  consistency and frontend performance.
+* Bad, because we still need to publish the govuk-frontend webjar on each new release of govuk-frontend
+* Bad, because the inlined images may become out of date.
```

Comment:
might not be worth the effort, but guess we could have a script that ran periodically somewhere that compared the hashes of the images in this repo and the govuk-frontend version we're targeting and created a PR for us when it changed with the updated image - not to say lets do that but just to take teeth out of this as a blocker